Update the doc of the IRC bridge integration. (#870)
zulip/python-zulip-api: zulip/integrations/bridge_with_irc/doc.md

# Zulip IRC Integration

Mirror an IRC channel in Zulip!

{start_tabs}

1.  {!download-python-bindings.md!}

1.  {!install-requirements.md!}

1.  Register a nick that ends with the suffix `_zulip` on your IRC server.

1.  {!create-a-generic-bot.md!}

1.  Download the `zuliprc` configuration file of your bot by clicking the
    download (<i class="fa fa-download"></i>) icon under the bot's name, and
    save to `~/.zuliprc`.

> 1.  [Subscribe the bot][subscribe-channels] to the Zulip channel that will
>     contain the mirror.

Reviewer: ```suggestion
1.  [Subscribe the bot][subscribe-channels] to the Zulip channel where IRC messages should be mirrored.
```

1.  Download the `zuliprc` configuration file of your bot by clicking the
    download (<i class="fa fa-download"></i>) icon under the bot's name, and
    save to `~/.zuliprc`.

1.  [Subscribe the bot][subscribe-channels] to the Zulip channel where IRC
    messages should be mirrored.

1.  Begin mirroring messages between the IRC channel and the Zulip channel
    by running the `irc-mirror.py` script with the
    [required command-line arguments](#required-arguments). Use the
    [optional arguments](#optional-arguments) to configure the mirroring
    behavior.

    Here's an example command that mirrors messages between the
    **#python-mypy** channel on the `irc.freenode.net` server and the "mypy"
    topic on the **#irc-discussions** channel on Zulip:

    ```
    python {{ integration_path }}/irc-mirror.py \
    --irc-server=irc.freenode.net --channel='#python-mypy' --nick-prefix=irc_mirror \
    --stream='irc-discussions' --topic='mypy'
    ```

1.  Messages will be mirrored only when the terminal session with the bot
    remains open. Consider using `screen` to run the bot in the background.
    You can restart the integration at any time by re-running the

>     `irc-mirror.py` script.

Reviewer 2: `screen` is a weird way to do it. We should suggest `supervisor` as a tool intended for keeping a job running.
